scripts/install_dev_server.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
The core difference is that I set up a lot of things as symlinks instead of installed files so that as you edit them in /opt/grr they take effect immediately without rerunning server install stuff.  It also serves as a wrapper for installing deps passing the specific requirements.txt.  

Some of the further reasoning is that with symlinks when the server repacks the clients they will repack to the host machine and can then be referenced for spinning up test clients to talk to the dev server.